Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM rigetti/forest:3.0.0
+
+# install jupyter notebook and jupyter lab
+RUN pip install --no-cache-dir notebook jupyterlab qiskit-rigetti qiskit[visualization]
```

Comment:
Do you think we should pin/constrain the `qiskit-rigetti` install? I believe binder will only rebuild the image if needed. Though since it's out of our control, it might be good to pin

Reply:
yeah - what do you think about pinning using a requirements file instead of inline?

Reply:
Sounds good. Will do 👍